scripts/ci-build-azure-ccm.sh

```diff
@@ -36,22 +36,41 @@ export CCM_IMAGE_NAME=azure-cloud-controller-manager
 export CNM_IMAGE_NAME=azure-cloud-node-manager
 declare -a IMAGES=("${CCM_IMAGE_NAME}" "${CNM_IMAGE_NAME}")
 
+SUPPORT_WINDOWS_SERVER_VERSION=("windows-2022" "windows-20H2" "windows-2019" "windows-2004")
+SUPPORT_WINDOWS_VERSION=("ltsc2022" "20H2" "1809" "2004")
+WINDOWS_VERSION=1809
+
 setup() {
     AZURE_CLOUD_PROVIDER_ROOT="$(go env GOPATH)/src/sigs.k8s.io/cloud-provider-azure"
     export AZURE_CLOUD_PROVIDER_ROOT
     # the azure-cloud-provider repo expects IMAGE_REGISTRY.
     export IMAGE_REGISTRY=${REGISTRY}
     pushd "${AZURE_CLOUD_PROVIDER_ROOT}" && IMAGE_TAG=$(git rev-parse --short=7 HEAD) && export IMAGE_TAG && popd
     echo "Image Tag is ${IMAGE_TAG}"
+
+    if [[ -n "${WINDOWS_SERVER_VERSION:-}" ]]; then
+        EXIST_FLAG=false
+        for i in "${!SUPPORT_WINDOWS_SERVER_VERSION[@]}"; do
+            if [[ ${WINDOWS_SERVER_VERSION} == ${SUPPORT_WINDOWS_SERVER_VERSION[$i]} ]]; then
+                WINDOWS_VERSION=${SUPPORT_WINDOWS_VERSION[$i]}
+                EXIST_FLAG=true
+                break
+            fi
+        done
+        if [[ ${EXIST_FLAG} == "false" ]]; then
+            echo "Not support windows version: ${WINDOWS_SERVER_VERSION}"
+            exit 1
+        fi
+    fi
 }
 
 main() {
-    if [[ "$(can_reuse_artifacts)" == "false" ]]; then
+    if [[ "$(can_reuse_artifacts)" =~ "false" ]]; then
         echo "Build Linux Azure amd64 cloud controller manager"
         make -C "${AZURE_CLOUD_PROVIDER_ROOT}" build-ccm-image-amd64 push-ccm-image-amd64
         if [[ -n "${TEST_WINDOWS:-}" ]]; then
-            echo "Building Linux amd64 and Windows ltsc2022 amd64 cloud node managers"
-            make -C "${AZURE_CLOUD_PROVIDER_ROOT}" build-node-image-linux-amd64 push-node-image-linux-amd64 push-node-image-windows-ltsc2022-amd64 manifest-node-manager-image-windows-ltsc2022-amd64
+            echo "Building Linux amd64 and Windows ${WINDOWS_VERSION} amd64 cloud node managers"
```

Comment:
```suggestion
            echo "Building Linux amd64 and Windows ${WINDOWS_IMAGE_VERSION} amd64 cloud node managers"
```

Reply:
Thanks for advice again! I have changed the code. If there is any room for improvement, tell me.